GitHub-Test-DHprojects/hammer_gitTest4.md

```diff
@@ -0,0 +1,12 @@
+# The Eldritch Project
+## Feedback
+
+It's quite an interesting topic to tackle and one that can take some time to seriously analyze due to the dated style of writing despite being one closer to modern times.
+
+
+One of the problems I encountered immediately looking into the text analysis was the very subjective nature of the word labeling. This is generally just due to some words having different connotations when used in different contexts, I find this heavily applies to the "Gothic" label as many of the words within it can be easily applied into "other" or don't have much that relates to it aside from the general nature vaguely relating to the topic. It's more that these words can be found frequently in the Gothic genre rather than actually being of that nature.
+
+
+The style of the website leaves a bit to be desired and I believe can be capitalized on quite a bit. The heavy usage of dark background contrasting with the bright reds and white of the text can be quite straining on the eyes to read and generally makes it more difficult to follow. I would personally recommend a style more akin to an actual page of a book and focus on that, instead of trying to get across the dark atmosphere through dark green on top of blinding whites and reds.
```

Comment:
Yeah I really did not like the layout. There werent many titles and the way the text was organized on the page was not quite right. And the font on the graph was hard to read.